platform-core/platform-common/pom.xml

```diff
@@ -60,6 +60,11 @@
 			<artifactId>commons-lang3</artifactId>
 			<version>3.3.2</version>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
```

Comment:
Please add test scope.

Reply:
Removed the dependency. It was not required.